### PR TITLE
Kaguya MI template

### PR DESCRIPTION
**Pull request by amystamile-usgs**
## Description

Completed Kaguya MI template

## How Has This Been Tested?

pvldiff and cubediff against kaguyami2isis test data.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [ ] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/appdata/import/KaguyaMI.tpl [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    MissionPhaseName             = {{ MISSION_PHASE_NAME.Value }}
+    RevolutionNumber             = {{ REVOLUTION_NUMBER.Value }}
+    StripSequenceNumber          = {{ STRIP_SEQUENCE_NUMBER.Value }}
+    SceneSequenceNumber          = {{ SCENE_SEQUENCE_NUMBER.Value }}
+    UpperLeftDaytimeFlag         = {{ UPPER_LEFT_DAYTIME_FLAG.Value }}
+    UpperRightDaytimeFlag        = {{ UPPER_RIGHT_DAYTIME_FLAG.Value }}
+    LowerLeftDaytimeFlag         = {{ LOWER_LEFT_DAYTIME_FLAG.Value }}
+    LowerRightDaytimeFlag        = {{ LOWER_RIGHT_DAYTIME_FLAG.Value }}
+    {% set detstatus=DETECTOR_STATUS.Value %}
+    DetectorStatus               = ({% for det in detstatus %}
```
Comment (scsides) [inline isis/appdata/import/KaguyaMI.tpl]: Have we tried using the inja join command for these?

**Review by scsides — COMMENTED** [review]
If the answer to my question is yes, go ahead and merge

**Review by gsn9 — APPROVED** [review]

File: isis/appdata/import/KaguyaMI.tpl (hunk 1 above)
Comment (scsides) [reply isis/appdata/import/KaguyaMI.tpl]: Answer is yes. We should use the join instead of the loop

**Review by scsides — APPROVED** [review]

File: isis/appdata/import/DawnVIR.tpl [hunk 2]
```diff
@@ -77,36 +62,14 @@
...
-                    {% endfor %})
-    {% set origArray=QUBE.BAND_BIN.BAND_BIN_ORIGINAL_BAND.Value %}
-    OriginalBand = ({% for band in origArray %}
-                    {% if loop.is_last %}
-                    {{ band }}
-                    {% else %}
-                    {{ band }},
-                    {% endif %}
-                    {% endfor %})
+    Center       = ({{ join(QUBE.BAND_BIN.BAND_BIN_CENTER.Value, ",") }})
```
Comment (scsides) [inline isis/appdata/import/DawnVIR.tpl:70]: Yah, three in a row makes it look a lot better

**Review by scsides — APPROVED** [review]

**Pull request merged**